developer-friendly script for running integration tests locally

cellog opened the pull request:
# Description

This PR introduces a new script, which is designed to make it easy to run the integration tests locally in a docker context repeatedly. It starts by using `docker down` to remove any running instances of the unlock integration tests, mainly `unlock`, `ganache-integration`. This also removes networks (a critical step). Next, it builds (or re-builds) the docker images locally using local cache, and finally it runs the integration tests

# Issues

<!-- This PR should fix or reference at least one existing issue ID. Add or delete as appropriate. -->

Refs #1890 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

<!--
PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)
-->

julien51 commented:
Please add nore info inside the file itself about its purpose and how to use it ;)

cellog commented:
> Please add nore info inside the file itself about its purpose and how to use it ;)

done, PTAL

akeem commented on scripts/local-docker-integration-tests.sh:
```diff
@@ -0,0 +1,35 @@
+
+# if the integration test images are running, this ensures we remove any state
+# prior to attempting to run the tests. This line is critical!
+docker-compose -f $DOCKER_COMPOSE_FILE down --volumes
```
Is `volumes` needed here?

cellog replied:
I guess not, we don't define any do we? I'll pull that out

cellog commented:
> I worry about the duplicate information between this file and `integration-tests.sh`.
> Could we only have the strictly necessary stuff in here and then invoke integration-tests.sh as the last step?

sure, I'll look into that, good idea!

cellog commented:
ok @julien51 PTAL.

We ended up needing the environment variables in both scripts, so I extracted them out. The odd syntax:

```bash
. $REPO_ROOT/scripts/integration-envs.sh
```

is needed to source the env variables. It's not at all intuitive, so I added a comment above to help clarify.

cellog commented:
update: this approach does not work. I don't think it's worth the effort to remove duplication, but I will have it invoke the integration_tests.sh script
